### DISCUSSION TITLE
PriceVariantProduct issues

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore.Commerce#460] [Q&A] [answered]
Hi,
I installed Orchard Core Commerce and so I launched **Orchard Core Commerce - Content - Product** recipe for trying the **PriceVariantProduct** type.

<img width="470" alt="1" src="https://github.com/OrchardCMS/OrchardCore.Commerce/assets/61869142/3781bbb4-3abe-4efc-a14d-20ed2bf9a092">

As you can see on the screenshots below, I set the predefined settings **M**, **L**, **XL** and a default value of the **Size** field.

<img width="1587" alt="2" src="https://github.com/OrchardCMS/OrchardCore.Commerce/assets/61869142/37a11a1a-d7ad-4d36-a63d-c9b18d6dd3c2"> 
<img width="479" alt="3" src="https://github.com/OrchardCMS/OrchardCore.Commerce/assets/61869142/4789695b-a37e-4138-8403-dcff5c7ced59">

But, when I open the page for creating a **PriceVariantProduct**:
- a **Price** variant is written as **X-L Price** instead of **XL Price**
- a **Inventory** variant is written as **L Inventory** instead of **XL Inventory**

<img width="1588" alt="4" src="https://github.com/OrchardCMS/OrchardCore.Commerce/assets/61869142/a6dc9a44-1c79-4cd1-98c3-1fd5122d38b2">

Also, when I try to create a **PriceVariantProduct**, I receive the error message **SKU must not empty** even if I set **SKU=CLOTHING**.

<img width="1601" alt="5" src="https://github.com/OrchardCMS/OrchardCore.Commerce/assets/61869142/5b356d17-5e11-49d4-899f-b37ae847f9cc">

I get the same result setting **Medium**, **Large**, **Extra Large** and a default value of the Size field.
I have no problem setting **S**, **M**, **L** and a default value of the Size field.
How can I solve this issue?

**Comment by sarahelsaig** [comment, marked as answer]
In the future please use issues for bugs, not discussions. Moved to https://github.com/OrchardCMS/OrchardCore.Commerce/issues/461
